architecture/core_concepts/pods_and_services.adoc

```diff
@@ -141,15 +137,14 @@ are filled by OpenShift automatically after the pod is created and
 its lifecycle begins. The
 xref:../../rest_api/kubernetes_v1.adoc#rest-api-kubernetes-v1[Kubernetes API documentation]
 has complete details of the pod REST API object attributes, and the
-link:https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/user-guide/pods.md[Kubernetes pod documentation]
+link:https://github.com/kubernetes/kubernetes/blob/release-1.0/docs/user-guide/pods.md[Kubernetes pod documentation]
```

Comment:
Is release-1.0 valid for all of these collections? Shouldn't it be different for each OCP version? (All places)